nats-base-client/jsclient.ts

```diff
@@ -92,6 +93,9 @@ class ViewsImpl implements Views {
   kv(name: string, opts: Partial<KvOptions> = {}): Promise<KV> {
     return Bucket.create(this.js, name, opts);
   }
+  bind(name: string, codec?: KvCodecs): Promise<KV> {
```

Comment:
this needs to be triggered by an option in KvOptions. - the views interface should simply be the raw functionality - kv, object store, etc.

Reply:
@aricart so basically additional "bind" option in KvOptions?

Reply:
yes!

Reply:
fixed.

went with `bindOnly` as thats more self-explenatory and clear IMO.